functests/4_latency/latency.go

```diff
@@ -164,7 +167,8 @@ var _ = Describe("[performance] Latency Test", func() {
 				Expect(curr < maximumLatency).To(BeTrue(), "The current latency %d is bigger than the expected one %d : \n %s", curr, maximumLatency, logFileContent)
 
 			}
-			testlog.Info(logFileContent)
+			//Use Println here so that this output will be displayed upon executing the test binary
+			fmt.Println(logFileContent)
```

Comment:
so this is shown inconditionally now? I wonder if this can cause regressions.

Reply:
Yes indeed, it will be printed unconditionally 
If we want to change it back to the original method we can, & then I think that we can workaround it in a way to save the output in a file and verify it from that file. Do you think we may face a complication with this solution?

Reply:
not sure, if test pass (but all of them including the intended usage with cnf-tests) I'm ok with showing unconditionally.

Reply:
I am not sure I understand what you mean

Reply:
nevermind :) I mean that unless we know this breaks something, let's keep it like you wrote it.
I don't think this is going to break anything, though.

Reply:
ok, will keep an eye on that. Thank you!